src/home/index.html

```diff
@@ -8,268 +8,308 @@
+              </li>
+              <li class="navbar-menu-item">
+                <a href="#">Projects</a>
+              </li>
+              <li class="navbar-menu-item">
+                <a href="#">Contact</a>
+              </li>
+            </ul>
+          </div>
         </div>
-      </section>
+        <nav class="mobile">
+          <img src="../assert/images/toggle.png" alt="Toggle" />
+        </nav>
+      </nav>
+      <header class="header">
+        <section class="headline">
+          <div class="text-home">
+            <h1 class="heading">I am Mwafrika Glad to see you!</h1>
+            <h3 class="description">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </h3>
+            <h3 class="subdescription">LET'S CONNECT</h3>
+          </div>
 
-      <section class="tonic">
-        <img src="../assert/images/tonic.png" alt="" />
-        <h2>Tonic</h2>
-        <ul class="tonic-list">
-          <li class="black">CANOPY</li>
-          <li class="dot"></li>
-          <li class="gray">Back End Dev</li>
-          <li class="dot"></li>
-          <li class="gray">2015</li>
-        </ul>
-        <h3 class="tonicDesc">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </h3>
-        <ul class="skills">
-          <li>html</li>
-          <li>css</li>
-          <li>javascript</li>
-        </ul>
-        <div class="btn">
-          <button type="button" class="btn-tonic">See Project</button>
+          <div class="social-container">
+            <ul>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/twitter.png" alt="Twitter" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/Linkedin.png" alt="LinkedIn" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/m.png" alt="Medium" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/github.png" alt="Github" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/fingers.png" alt="Finger" />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </section>
+      </header>
+      <section class="works">
+        <div class="card-1">
+          <div class="card-1-2">
+            <div class="image-container"></div>
+          </div>
+          <div class="card-1-1">

```

Comment:
- [ ] Please ensure that your styling in the desktop version matches the Figma design in the card section.

|Figma Design|Your Design|
|---|---|
|![image](https://user-images.githubusercontent.com/18730106/149633510-82fba283-539c-46ca-b500-9edfefe88229.png)|![image](https://user-images.githubusercontent.com/18730106/149633575-7b5ff989-2be5-417d-9ea4-39397f3b6ed0.png)|

Reply:
Done already!